docs/how-to/deploy_charmed_magma_access_gateway.md

```diff
@@ -25,7 +25,7 @@ The Access Gateway must be installed on an Ubuntu machine with the following spe
     
     Deploy Magma Access Gateway:
     ``` bash
-    juju deploy magma-access-gateway-operator --config sgi=enp0s1 --config s1=enp0s2
+    juju deploy magma-access-gateway-operator --config sgi=enp0s1 --config s1=enp0s2 --channel=1.6/stable
     ```
     
```

Comment:
At the end of "Option 2: Static network configuration" we should change it as well.
(I can't comment in that place, as there are no changes)
```suggestion
juju deploy magma-access-gateway-operator --config agw_config.yaml --channel=1.6/stable
```

Reply:
✅